bindings/javascript/http/batch.js

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
nit: file name index.js?

Reply:
Ok I guess we could go to language defaults across the board, huh?

Reply:
Yes! So that the users will know where the code is.

Reply:
Ok doing for all!